conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureAuthorizationExtractor.java

```diff
@@ -38,6 +40,16 @@
  */
 final class ConjureAuthorizationExtractor implements AuthorizationExtractor {
 
+    private static final String USER_ID_KEY = "userId";
+    private static final String SESSION_ID_KEY = "sessionId";
+    private static final String TOKEN_ID_KEY = "tokenId";
+    private static Consumer<String> sessionIdSetter = sessionId -> MDC.put(SESSION_ID_KEY, sessionId);
+    private static Consumer<String> tokenIdSetter = tokenId -> MDC.put(TOKEN_ID_KEY, tokenId);
+    private static final ErrorType MISSING_CREDENTIAL_ERROR_TYPE = ErrorType.create(
+            ErrorType.Code.UNAUTHORIZED, "Conjure:MissingCredential");
+    private static final ErrorType MALFORMED_CREDENTIAL_ERROR_TYPE = ErrorType.create(
+            ErrorType.Code.UNAUTHORIZED, "Conjure:MalformedCredential");
```

Comment:
This might be a minor nit but I never heard this word being used in the singular in the context of authorization.
Thoughts on calling it `credentials"?

Reply:
Good call. We could also use `Token` for conformity with https://github.com/palantir/auth-tokens